db/migrate/20160329231848_create_thredded.rb

```diff
@@ -219,6 +216,23 @@ def change
               order: { created_at: :desc },
               name:  :index_thredded_moderation_records_for_display
     end
+
+    create_table :thredded_notifications_for_private_topics do |t|
+      t.integer :user_id, null: false
+      t.string :notifier_key, null: false
+      t.boolean :wants, default: true, null: false
+    end
+    create_table :thredded_notifications_for_followed_topics do |t|
+      t.integer :user_id, null: false
+      t.string :notifier_key, null: false
+      t.boolean :wants, default: true, null: false
+    end
+    create_table :thredded_messageboard_notifications_for_followed_topics do |t|
+      t.integer :user_id, null: false
+      t.integer :messageboard_id, null: false
+      t.string :notifier_key, null: false
+      t.boolean :wants, default: true, null: false
+    end
```

Comment:
Perhaps it would make sense to enforce integrity with unique indices, e.g.:

```ruby
  t.index [:user_id, :messageboard_id, :notifier_key],
          name: 'thredded_messageboard_notifications_for_followed_topics_unique'.
          unique: true
``